modules/core/src/bitgo.ts

```diff
@@ -942,29 +937,6 @@ export class BitGo {
     return true;
   }
 
-  /**
-   * Construct an ECDH secret from a private key and other user's public key
-   */
-  getECDHSecret({ otherPubKeyHex, eckey }: GetEcdhSecretOptions): string {
```

Comment:
Could you please keep this function on the `bitgo` object as a pass through to the new one in `ecdh.ts` for the moment (and mark as `@deprecated`)? There are at least two places in the UI where this is called directly.

We can remove in a future major version.

Reply:
ouch, forgot to check there